### PR TITLE
layout_calendar

### PR DESCRIPTION
**Pull request by oksanatytanych**
- [DEMO LINK](https://oksanatytanych.github.io/layout_calendar/)
- [TEST REPORT LINK](https://oksanatytanych.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <section class="calendar calendar__start-day--tue calendar__month-length--30">
```
Comment (DiDemkiv) [inline src/index.html]: better use modifiers of the calendar ("--")
```suggestion
    <section class="calendar calendar--start-day--tue calendar--month-length--30">
```

File: src/styles/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  display: flex;
+  flex-wrap: wrap;
+
+  width: 7 * $square100 + 6 * $space-between +20px;
+
+  padding: 10px;
+  gap: $space-between;
+
+  
+
```
Comment (DiDemkiv) [inline src/styles/calendar.scss]: Let's display the calendar in the center of the screen. 
For example, you can create a separate block for the body and consider using flex on it with align-items: center.

File: src/styles/normalize.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+
+}
+
+.body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
```
Comment (DiDemkiv) [inline src/styles/normalize.scss]: create a separate block for body and add height: 100vh

File: src/styles/normalize.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+  margin: 0;
```
Comment (DiDemkiv) [inline src/styles/normalize.scss]: let's move it to reset.scss

File: src/styles/calendar.scss (hunk 2 above)
Comment (DiDemkiv) [inline src/styles/calendar.scss]:
```suggestion
```

**Review by DiDemkiv — CHANGES_REQUESTED** [review]
Small mistakes, but otherwise looks good to me! 🔥 Good job

File: src/styles/body.scss [hunk 5]
```diff
@@ -0,0 +1,8 @@
+.body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+
+  margin: 0;
```
Comment (DiDemkiv) [inline src/styles/body.scss]: it's wrong to use margin in bem blocks
let's move it to reset.scss :)

**Review by DiDemkiv — CHANGES_REQUESTED** [review]
Structurize your files. All blocks should be in the blocks folder. Don't forget about the utils folder with "normalize.scss", "reset.scss" and "variables.scss" files
msg541127377-601223

**Review by DiDemkiv — APPROVED** [review]

File: src/styles/blocks/calendar.scss [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  flex-wrap: wrap;
+
+  width: 7 * $square100 + 6 * $space-between +20px;
+
+  padding: 10px;
+  gap: $space-between;
+
+  &__day {
+    width: $square100;
+    height: $square100;
```
Comment (Ihor-Karpyn) [inline src/styles/blocks/calendar.scss:12]: mixin square

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**